pyproject.toml

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright 2014 Climate Forecasting Unit, IC3
+# Copyright 2014-2025 Barcelona Supercomputing Center (http://www.bsc.es/) 
```

Comment:
https? :grimacing: Avoids a redirect and some linters look for http and then you have to whitelist the ones that are not really links used/clickable-links :pray: